### PR TITLE
Adds some image attribtuions

### PR DESCRIPTION
**Pull request by bennett000**

### CODE REVIEW

File: handout/images/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- app-screenshot.png - Screenshot of our plunker
+- change-detection.jpg -  https://pixabay.com/en/coins-handful-russia-ruble-kopek-650779/ - cli-folder-setup.png - Screenshot of files
+- di-tree.png - Michael made this
+- es6-typescript.png - Michael made this
+- flat-map.png - Observable docs? Open source?
+- migration.jpg - https://pixabay.com/en/great-migration-africa-animal-1021460/
+- pipes.jpg - https://pixabay.com/en/pipe-plumbing-connection-pipeline-406906/
+- simple-output.png - Screenshot of our output
+- tools.jpg - http://commons.wikimedia.org/wiki/File:Hand_tools.jpg
+- uluru.jpg - https://commons.wikimedia.org/wiki/File:Ayers_Rock_Uluru.jpg
```
Comment (SethDavenport) [inline handout/images/README.md]: @bennett000 for stuff that we made internally/you made, should we license them with CC with similar captions?

Comment (SethDavenport) [inline handout/images/README.md]: Also is this file intended to stay here?

Comment (SethDavenport) [inline handout/images/README.md]: Screenshots of our plnkrs probably don't need formal attributions?

Comment (SethDavenport) [inline handout/images/README.md]: Let's at least get in what we do know, and keep the list of what we don't know internal while we hunt things down I would say.

**Comment by bennett000** [comment]
@SethDavenport at standup we informally agreed to not caption our own work.

There might be a tiny bit of utility in keeping the `README.md` in the images folder but it would be cleaner to delete it.  I'll defer to anyone with an opinion on this.

File: handout/images/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Image Attributions
+
+- advanced-component.jpg - https://commons.wikimedia.org/wiki/File:GB_Network_PCI_Card.jpg
+- angular-di.jpg *UNCERTAIN* Cannot Find Image
+- angular1-vs-angular2.jpg - *UNCERTAIN* Ask David
```
Comment (barretodavid) [inline handout/images/README.md]: @bennett000 I made that one

**Comment by bennett000** [comment]
Self :shipit:, other editors have verbally shipped it.
